.devcontainer/linux/devcontainer.json

```diff
@@ -0,0 +1,12 @@
+// vi: ft=jsonc
+{
+  "build": { "dockerfile": "Dockerfile", "context": "../" },
+  "customizations": {
+    "vscode": {
+      "extensions": ["dbaeumer.vscode-eslint"]
+    }
+  },
+  "runArgs": ["--userns=keep-id"],
+
+  "forwardPorts": [3000]
+}
```

Comment:
@coderabbitai write documentation on how developers utilize devcontainers in the root README

Reply:
Please make this a code suggestion for easy inclusion